manta.wdl

```diff
@@ -20,7 +20,63 @@ version 1.0
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import "common.wdl"
+task Germline {
+    input {
+        File bamFile
+        File bamIndex
+        File referenceFasta
+        File referenceFastaFai
+        String runDir
```

Comment:
It is best to give runDir a default. If a task runs in the cloud you should not have to worry about this.